libs/tdb/files/tdb.cache.txt

```diff
@@ -0,0 +1,36 @@
+Checking simple C program: OK
```

Comment:
The package uses the 'waf'[https://waf.io/](url) as its build system which need cache.txt (required by cross-answers option) file when cross-compile. Waf seems it can not figure out some of the options, so we need to preset them manually.  Samba 4 does this as well. [https://git.busybox.net/buildroot/tree/package/samba4](url)